### DISCUSSION TITLE
hey

### DISCUSSION THREAD

**Discussion by devmailar** [community/community#13451] [Other Feature Feedback, Questions, & Ideas] [answered]
hello guys

**Comment by suprstarrd** [comment]
this is the best feedback i have ever seen. truly outstanding

**Reply by devmailar** [reply]
Thank you

**Comment by suprstarrd** [comment, marked as answer]
i see you're a student from your bio! consider grabbing the [Student Developer Pack](https://education.github.com/pack)

also this is the Feed Feedback section so perhaps give some words on the new for you section

**Reply by nokiae0106** [reply]
Gg

**Reply by nokiae0106** [reply]
> i see you're a student from your bio! consider grabbing the [Student Developer Pack](https://education.github.com/pack)
> 
> also this is the Feed Feedback section so perhaps give some words on the new for you section
